test/parallel/test-crypto-sec-level.js

```diff
@@ -0,0 +1,11 @@
+// Flags: --expose-internals
+'use strict';
+
+const common = require('../common');
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+
+const assert = require('assert');
+
+const secLevel = require('internal/crypto/util').getOpenSSLSecLevel();
+assert.ok(secLevel >= 0 && secLevel <= 5);
```

Comment:
A comment explaining these numbers and where they come from would be helpful for future generations coming into this code later.